doc-content/enterprise-only/admin-and-config/business-central-editing-data-sources-proc.adoc

```diff
@@ -0,0 +1,70 @@
+[id='managing-business-central-data-sources-con']
+= Managing data sources
+
+{CENTRAL} provides data source management which gives you the ability of defining data sources for accessing a database. These data sources are then used by other {CENTRAL} components such as the data sets. A database driver, on the other hand, is used for enabling communication between a data source and the targeted database.
+
+From the *Data Source Authoring* page you can add data sources and database drivers to {CENTRAL}.
+
+[float]
+==== Adding a data source to {CENTRAL}
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Data Sources*. The *Data Sources Authoring* page opens.
+. In the *DataSource Explorer* pane, click on *Add DataSource*. The *New data source* window opens.
+. In the *New data source* window, enter the *Name*, *Connection URL*, *User*, *Password*, and the *Driver* of the data source.
```

Comment:
This step is unclear. We have introduced two terms in the introduction: a `data source` and `database`.
In Business Central, we are creating a data source construct which will communicate with a particular database.
This means that the step here needs to make the same distinction as well:
`In the *New data source* window, enter data source *Name*, database *Connection URL*, *User* and *Password*, and the *Driver* which will be used to establish the connection with the database.`

Reply:
Re-phrased the step.